index.html

```diff
@@ -0,0 +1,125 @@
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </section> 
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="social">
+            <div class="social-media">
+            <ul class="social-media-icon">
+                <li><a href="#"><img src="./assets/icons/github.png" alt="Github"></a></li>
+                <li><a href="#"><img src="./assets/icons/linkedin.png" alt="Linkedin"></a></li>
+                <li><a href="#"><img src="./assets/icons/Vector.png" alt="Angellist"></a></li>
+                <li><a href="#"><img src="./assets/icons/twitter.png" alt="Twitter"></a></li>
+                <li><a href="#"><img src="./assets/icons/medium.png" alt="Medium"></a></li>
+            </ul>
+                <a class="arrow" href="#"><img src="./assets/icons/element-right.png" alt="arrow"></a>   
+            </div>   
+            <div class="experience-title">
+                <h2>MY RECENT <br> WORK</h2>
+            </div> 
+             
+        </section>
+        
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>    
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>  
+                <li>Ruby</li>
+            </ul>
+
+            <button type="button" class="project-button">See Project</button>
+
+        </section>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css <img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>          
+                <li>bootstrap <img src="./assets/images/vert-line.png" alt="vertical line"></li>         
+                <li>Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+
```

Comment:
- [ ] Please delete the extra blank line here
_For more details about this requirement, please see the comment above._